python-avd/pyavd/_eos_designs/schema/__init__.py

```
@@ -42452,6 +42513,7 @@ def __init__(
                     private_vlan: bool | UndefinedType = Undefined,
                     sflow: bool | UndefinedType = Undefined,
                     sflow_subinterfaces: bool | UndefinedType = Undefined,
+                    sflow_subinterfaces_future: bool | None | UndefinedType = Undefined,
```

Ruff (0.16.0)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)